pybedtools_parser.py

```diff
@@ -0,0 +1,28 @@
+import sys
+
+
+def parsing_bed(bed_file):
+    parsed_data = list()
+    with open(bed_file) as input_handle:
+        for line in input_handle:
+            line_elements = line.split()
+            # Ignore annotation lines.
+            if any(i in line_elements for i in ("#", "browser", "track")):
+                continue
+            # Check if a non-annotation line if valid.
+            elif len(line_elements) < 3:
+                sys.exit("Program expects a valid BED file with at least 3 columns (chromosome, start and end).\n"
```

Comment:
IMHO `sys.exit` is not the best way. Some kind of error is more suitable.
Python has multiline strings, sot there's no need in `\n`